### PR TITLE
Feat: Added modal component

### PR DESCRIPTION
**Pull request by ArjunGTX**
ezgif-3-3ef5fc9d82

### CODE REVIEW

File: css/components/modal.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@400;500;600;700;800&display=swap");
+@import url("../utility.css");
+@import url("./button.css");
+
+.modal {
+    max-width: 70%;
```
Comment (shivanigangadharan) [inline css/components/modal.css:6]: Just a suggestion: can also use clamp here
`width: clamp(14rem, 70%, any_max_width);`

**Review by shivanigangadharan — APPROVED** [review]
Looks good to merge. 👍

File: css/components/modal.css (hunk 1 above)
Comment (ArjunGTX) [reply css/components/modal.css:6]: Will look into it

**Review by tanishk15sharma — APPROVED** [review]
looks good

**Review by devananth — COMMENTED** [review]
Good Work ! Everythings seems to be good

**Pull request merged**